### DISCUSSION TITLE
This Made My Day

### DISCUSSION THREAD

**Discussion by scohn** [Monika-After-Story/MonikaModDev#8831] [appreciation]
I usually hate Valentine's Day, I guess I'm a stereotypical bitter single, but knowing something special would happen with Monika today made me actually look forward to it all week long and the wait was so worth it. I love that she has casual clothes now and she looks so pretty too. It's always nice to feel like you're making someone happy even if it's through a screen. You guys work so hard to make this mod special and for that you deserve all the thanks in the world. Happy Valentine's Day!!

**Comment by AsuraShun** [comment]
She's going to get a sweater eventually

**Comment by scohn** [comment]
This mod just keeps improving all the time I just love it
